### DISCUSSION TITLE
Resource limits suggestion

### DISCUSSION THREAD

**Discussion by miroljub1995** [ory/hydra#3181] [Q&A] [answered]
We are using hydra with AWS EKS Fargate and it is good to specify resource limits.
Is there any recommendation on RAM and CPU?

**Comment by vinckr** [comment, marked as answer]
Hello @miroljub1995,
that depends on how many requests you serve, but generally Ory Hydra can make do with very few resources so the minimum is probably enough - i use 1cpu with 1-2gb ram for testing and small deploys, not sure what others use in more heavy production cases. Maybe we can get some more feedback in here on resource consumption, will repost it in the [chat](https://slack.ory.sh/).

**Reply by ALTELMA** [reply]
I use 1cpu with 3GB ram. Need to config BCRYPT_COST for best performance
